fre/app/generate_time_averages/frepytoolsTimeAverager.py

```diff
@@ -14,8 +14,19 @@ class inheriting from abstract base class timeAverager
     '''
 
     def generate_timavg(self, infile=None, outfile=None):
-        ''' frepytools approach in a python-native manner.
-        deliberately avoids pre-packaged routines. '''
+
+        """
+        frepytools approach in a python-native manner.
+        deliberately avoids pre-packaged routines.
+
+        :param self: 
+        :param infile: path to history file, or list of paths
+        :type infile: str, list
+        :param outfile: path to where output file should be stored
+        :type outfile: str
+        :param self: time scale for climatology. Accepted variables vary based on pkg ('all','seas','month') 
+        :type self: str, defaults to 'all'. Inherted from generate_time_averages.py
+        """
```

Comment:
Add the returns 1 and 0 similarly to the other 2 files.  But the description of a return value of 1 will be unique to describe the cases where we would return 1 (lines 39 and 68)